dolphinscheduler-ui/src/locales/modules/zh_CN.ts

```diff
@@ -2,6 +2,7 @@
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
+    mlflowProjectRepository: '运行仓库',
```

Comment:
Licensed header should not included this.

Reply:
Thanks for reviewing, I have fixed it.